src/main/scala/io/iohk/ethereum/crypto/ECDSASignature.scala

```diff
@@ -48,6 +56,25 @@ object ECDSASignature {
     ECDSASignature(r, s, pointSign)
   }
 
+  def sign2(message: Array[Byte], keyPair: AsymmetricCipherKeyPair, chainId: Option[Byte] = None): ECDSASignature = {
```

Comment:
What's this? I'm not seeing it used anywhere?

Reply:
sorry I used it to generate signature with random k, will remove it